examples/access_traits/example_cuda_access_traits.cpp

```diff
@@ -72,9 +72,8 @@ int main(int argc, char *argv[])
     Kokkos::Cuda cuda{stream};
     ArborX::BVH<Kokkos::CudaSpace> bvh{cuda, PointCloud{d_a, d_a, d_a, N}};
 
-    using DeviceType = Kokkos::Cuda::device_type;
-    Kokkos::View<int *, DeviceType> indices("indices", 0);
-    Kokkos::View<int *, DeviceType> offset("offset", 0);
+    Kokkos::View<int *, Kokkos::Cuda> indices("indices", 0);
```

Comment:
No I was not planning on reverting.

Reply:
Why not? I thought we wanted to teach users to template their views on memory space, or device type, and then pass execution space? I think this teaches them a wrong lesson, maybe that Cuda automatically means CudaSpace, though it could be CudaUVM.

Reply:
Both versions rely on the default memory space for CUDA

Reply:
I agree that it doesn't make a difference but conceptually, it is easier to understand for people that don't know Kokkos if we say that you need to use a `MemorySpace` for the `View` (data).

Reply:
Updated